.github/workflows/nightly.yml

```diff
@@ -54,3 +55,52 @@ jobs:
 
       - name: Valgrind Checks
         run: python3 ./src/scripts/ci_build.py --cc=gcc --make-tool=make valgrind-full
+
+  tls_anvil_server_test:
+    name: "TLS-Anvil - Botan Server Test"
+
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Fetch Botan Repository
+        uses: actions/checkout@v3
+
+      - name: Setup Build Agent
+        uses: ./.github/actions/setup-build-agent
+        with:
+          target: static
+          cache-key: linux-clang-x86_64-static
```

Comment:
This cache key seems wrong since we are using gcc to build

I think if we use the `linux-gcc-x86_64-static` key, then we can share caches with this build and the normal static library GCC CI build.